macros/public/all_evm_chains.sql

```diff
@@ -1,3 +1,3 @@
 {% macro all_evm_chains() %}
-   {{ return(array('ethereum', 'optimism', 'arbitrum', 'avalanche_c', 'polygon', 'bnb', 'gnosis', 'fantom')) }}
+   {{ return(array('ethereum', 'optimism', 'arbitrum', 'avalanche_c', 'polygon', 'bnb', 'gnosis', 'fantom', 'base', 'bitcoin', 'celo', 'goerli', 'solana')) }}
```

Comment:
ah, this is `all_evm_chains`, so i don't think we'd include bitcoin/solana? not sure about celo

Reply:
the other three macros are fine, but this could cause impact downstream in spellbook

Reply:
i would just ignore this file in this PR.

@MSilb7 is updating these macros here: https://github.com/duneanalytics/spellbook/pull/3931/files

Reply:
AH, you're right - I should not include them on DuneSQL either

Reply:
I'll drop the changes for it here.